src/cli/cmd/init/mod.rs

```diff
@@ -77,13 +82,14 @@ impl CommandExecute for InitSubcommand {
             .as_str()
             {
                 // MAYBE: find an enum-based approach to this
-                "23.05" => String::from("0.2305.*"),
-                "latest" => String::from("*"),
-                "unstable" => String::from("0.1.*"),
+                "23.05" => String::from(NIXPKGS_23_05),
```

Comment:
fwiw, I was more talking about the thing we're matching on, since it's used in more than 1 place (the list up above), and a typo leading to the fall-through branch would be unfortunate.